### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by thomasly** [lennylv/CAPLA#1]
Hi,
I found your work interesting and would like to test your model with some protein-ligand complexes generated in my own lab? How to prepare my protein data to get the pock file as in your training and test set? And how do I run a inference with your trained model? Thank you!

**Comment by divyavani01** [comment]
yeah i also wanted to try your model with my own data...how can i generate global and pocket files....can you plz help me out with this
